azurerm/resource_arm_storage_account.go

```diff
@@ -1022,6 +1211,136 @@ func expandStorageAccountBypass(networkRule map[string]interface{}) storage.Bypa
 	return storage.Bypass(strings.Join(bypassValues, ", "))
 }
 
+func expandQueueProperties(input []interface{}) (queues.StorageServiceProperties, error) {
+	properties := queues.StorageServiceProperties{}
+	if len(input) == 0 {
+		return properties, nil
+	}
+
+	attrs := input[0].(map[string]interface{})
+
+	properties.Cors = expandQueuePropertiesCors(attrs["cors_rule"].([]interface{}))
+	properties.Logging = expandQueuePropertiesLogging(attrs["logging"].([]interface{}))
+	minuteMetrics, err := expandQueuePropertiesMetrics(attrs["minute_metrics"].([]interface{}))
```

Comment:
This could be:
```suggestion
	properties.MinuteMetrics, err := expandQueuePropertiesMetrics(attrs["minute_metrics"].([]interface{}))
``` 🤔

Reply:
That'd be nice but I get an error when I try for it 
```
non-name properties.MinuteMetrics on left side of :=
```

Reply:
try: 
```suggestion
	properties.MinuteMetrics, err = expandQueuePropertiesMetrics(attrs["minute_metrics"].([]interface{}))
```